src/main/java/viper/gobra/frontend/GobraParserBase.java

```diff
@@ -0,0 +1,130 @@
+package viper.gobra.frontend;
+import java.util.List;
+import org.antlr.v4.runtime.*;
+import viper.gobra.frontend.GobraLexer;
+
+/**
+ * All parser methods that used in grammar (p, prev, notLineTerminator, etc.)
+ * should start with lower case char similar to parser rules.
+ */
+public abstract class GobraParserBase extends org.antlr.v4.runtime.Parser
```

Comment:
This is code used to handle line separators in the parser and is unique to the Go grammar. I'll add a comment that links to the original repo.
It's not correct though, I'm also working on fixing it, contributing that to upstream and then update it here after it's merged, but I think this has quite low priority since the old SemicolonPreprocessor fixes this too.